tools.js

```diff
@@ -573,6 +573,36 @@ module.exports = (function () {
 		return banlistTable;
 	};
 
+	Tools.prototype.getFullLearnset = function (pokemon) {
```

Comment:
<code>.getLearnset</code> would be a better name for consistency with the client.